frontend/src/components/Documentation.vue

```diff
@@ -395,98 +395,70 @@
-              </video>
-              <figcaption>Demo of the interaction partners tool</figcaption>
+            <figure id="interaction-partners-figure">
+              <img src="@/assets/interPart.jpg" alt="interaction partners example" />
+              <figcaption>Example of the Interaction Partners tool</figcaption>
             </figure>
-            <div>
-              <p>
-                For a given metabolite or gene, this tool (shown in the demo video) renders a
-                connected graph of the other metabolites and genes with which it interacts via
-                shared reaction(s). The metabolite or gene of interest is centered on the graph.
-                Connectivity is determined based on the reactions with which this metabolite or gene
-                is associated, i.e. an edge between two nodes may represent a connections that
-                occurs in multiple reactions. For medium-sized networks of interaction partners,
-                there is a prompt before generating the graph. For very large networks (e.g. for H
-                <sub>2</sub>
-                O) the graph will not be generated. One can left-click a node to display a context
-                menu with 3 options:
-              </p>
-              <ul>
-                <li>
-                  Load interaction partners: reload the interaction partners graph with the clicked
-                  node as the new central node.
-                </li>
-                <li>
-                  Expand interaction partners: add additional interaction partner nodes for the
-                  clicked node to the graph. Expanded interactions are represented with dashed
-                  lines.
-                </li>
-                <li>
-                  Highlight reaction: some nodes may be involved in many different reactions. Select
-                  a reaction from the list to show the other interaction partners associated with
-                  the selected reaction (other interaction partners will be grayed-out). The
-                  directionality of each edge is indicated as a triangle, or diamonds in case of
-                  reversible reactions. To remove the highlight, click on the
-                  <span><i class="fa fa-eraser"></i></span>
-                  eraser button at the top of the graph.
-                </li>
-              </ul>
-            </div>
+            <p>
+              For a given metabolite or gene, this tool (shown in the figure to the right) renders a
+              connected graph of the other metabolites and genes with which it interacts via shared
+              reaction(s). The metabolite or gene of interest is centered on the graph. Connectivity
+              is determined based on the reactions with which this metabolite or gene is associated,
+              i.e. an edge between two nodes may represent a connections that occurs in multiple
+              reactions.
```

Comment:
```suggestion
              For a given metabolite or gene, this tool (shown in the figure to the right) renders a
              connected graph of the other metabolites and genes that it interacts with via shared
              reaction(s). The metabolite or gene of interest is centered on the graph. An edge 
              between two nodes represent a connection that occurs via one or multiple reactions.
```

Reply:
@mihai-sysbio does this suggestion look good to you? If not, please suggest something else or write that we should keep the old version

Reply:
I think the text suggested by @inghylt is cleaner.

Reply:
I like the suggestion too!